### ISSUE TITLE
Order of import statements impacts memory?

### ISSUE THREAD

**Issue by ATMakersBill** [adafruit/circuitpython#1335]
The following code succeeds in running:
```
from adafruit_hid.keyboard import Keyboard
from adafruit_hid.keyboard import Keycode
import adafruit_rfm69
#code to do the work...
```

while the following fails:
```
import adafruit_rfm69
from adafruit_hid.keyboard import Keyboard
from adafruit_hid.keyboard import Keycode
#code to do the work...
```
I get the following error: 
```
Traceback (most recent call last):
  File "main.py", line 10, in <module>
  File "adafruit_hid/keyboard.py", line 35, in <module>
MemoryError: memory allocation failed, allocating 930 bytes
```

This is on an feather_m0_rfm69 using the latest master (4.0alpha) release on 11/14

**Comment by tannewt** [comment]
It can because the memory gets fragmented differently depending on allocation order and the size of objects. Loading mpy files uses less memory so its recommended to do that for files you aren't editing.

**Comment by ladyada** [comment]
@kattni this does come up once a while, please add it to our FAQ :)
https://learn.adafruit.com/circuitpython-essentials/circuitpython-expectations#frequently-asked-questions-18-8

**Comment by ATMakersBill** [comment]
Ok, I'm OK with that - Scott, are there MPY files in the standard bundle for HID?  I thought they were split out so you didn't have to load the whole thing - can they still be compiled?

**Comment by dhalbert** [comment]
@ATMakersBill If you're using 4.0-alpha, then it doesn't yet have the "swap `.frozen` and `.lib`" fix that was put in to 3.1.1 and later. So make sure the lib directory _doesn't_ have an `adafruit_hid` directory, if you've frozen in HID.

You might want to do these frozen builds from the 3.x branch instead, unless you need 4.0 (I can't remember.

**Comment by kattni** [comment]
Added to Expectations FAQ.
